lib/active_model/serializer/attributes.rb

```diff
@@ -14,10 +14,12 @@ module Attributes
 
         # Return the +attributes+ of +object+ as presented
         # by the serializer.
-        def attributes(requested_attrs = nil, reload = false)
+        def attributes(options = {}, reload = false)
+          requested_attrs = options[:only]
+          excepts = Array(options[:except])
```

Comment:
@noahsilas if you can add a test using only/except and if/unless, I'll accept the PR.

I see this being used for excluding attributes more often than including them. Such as excluding the created_at and updated_at fields.